pom.xml

```diff
@@ -741,7 +741,7 @@
         <module>gradle</module>
         <module>container</module>
         <module>server</module>
-        <module>examples</module>
+<!--        <module>examples</module>-->
```

Comment:
Why are we excluding the `examples` module?

Reply:
Mistakenly ended up in here.